modules/mobile/spec/request/payment_information_request_spec.rb

```diff
@@ -193,12 +193,12 @@
     context 'when the user does have an associated email address' do
       subject do
         VCR.use_cassette('lighthouse/direct_deposit/update/200_valid') do
-          put '/mobile/v0/payment-information/benefits', params: payment_info_request, headers:
```

Comment:
spec was returning an error without sis headers but no `expect` was being run so it wasn't getting caught.